docs/en/notes/faq.md

```diff
@@ -4,37 +4,19 @@ We list some common troubles faced by many users and their corresponding solutio
-|         0.22.0         |   mmcv-full>=1.4.4, \<=1.6.0   |       mmcls>=0.20.1, \<=1.0.0       |          Not required          |
-|         0.21.1         |   mmcv-full>=1.4.4, \<=1.6.0   |            Not required             |          Not required          |
-|         0.20.2         |  mmcv-full>=1.3.13, \<=1.6.0   |            Not required             |          Not required          |
-|         0.19.0         |  mmcv-full>=1.3.13, \<1.3.17   |            Not required             |          Not required          |
-|         0.18.0         |  mmcv-full>=1.3.13, \<1.3.17   |            Not required             |          Not required          |
-|         0.17.0         |   mmcv-full>=1.3.7, \<1.3.17   |            Not required             |          Not required          |
-|         0.16.0         |   mmcv-full>=1.3.7, \<1.3.17   |            Not required             |          Not required          |
-|         0.15.0         |   mmcv-full>=1.3.7, \<1.3.17   |            Not required             |          Not required          |
-|         0.14.1         |   mmcv-full>=1.3.7, \<1.3.17   |            Not required             |          Not required          |
-|         0.14.0         |   mmcv-full>=1.3.1, \<1.3.2    |            Not required             |          Not required          |
-|         0.13.0         |   mmcv-full>=1.3.1, \<1.3.2    |            Not required             |          Not required          |
-|         0.12.0         |   mmcv-full>=1.1.4, \<1.3.2    |            Not required             |          Not required          |
-|         0.11.0         |   mmcv-full>=1.1.4, \<1.3.0    |            Not required             |          Not required          |
-|         0.10.0         |   mmcv-full>=1.1.4, \<1.3.0    |            Not required             |          Not required          |
-|         0.9.0          |   mmcv-full>=1.1.4, \<1.3.0    |            Not required             |          Not required          |
-|         0.8.0          |   mmcv-full>=1.1.4, \<1.2.0    |            Not required             |          Not required          |
-|         0.7.0          |   mmcv-full>=1.1.2, \<1.2.0    |            Not required             |          Not required          |
-|         0.6.0          |   mmcv-full>=1.1.2, \<1.2.0    |            Not required             |          Not required          |
+| MMSegmentation version |          MMCV version          | MMEngine version  | MMClassification (optional) version | MMDetection (optional) version |
+| :--------------------: | :----------------------------: | :---------------: | :---------------------------------: | :----------------------------: |
+|     dev-1.x branch     |        mmcv >= 2.0.0rc4        | MMEngine >= 0.2.0 |           mmcls>=1.0.0rc0           | mmdet>=3.0.0rc4, \<=3.0.0rc5>  |
+|       1.x branch       |        mmcv >= 2.0.0rc4        | MMEngine >= 0.2.0 |           mmcls>=1.0.0rc0           | mmdet>=3.0.0rc4, \<=3.0.0rc5>  |
```

Comment:
```suggestion
|       1.x branch       |        mmcv == 2.0.0rc3        | MMEngine >= 0.1.0 |           mmcls>=1.0.0rc0           | mmdet>=3.0.0rc4, \<=3.0.0rc5>  |
```
as 1.x branch  updates  when mmseg release, I suggest modifying 1.x branch dependency in bump pr

Reply:
> as 1.x branch updates when mmseg release, I suggest modifying 1.x branch dependency in bump pr

get